src/view_page.cpp

```diff
@@ -1718,7 +1718,7 @@ void View::DrawSystemEditorialElement(DeviceContext *dc, EditorialElement *eleme
         assert(dynamic_cast<Choice *>(element) && (dynamic_cast<Choice *>(element)->GetLevel() == EDITORIAL_TOPLEVEL));
     }
     std::string elementStart;
-    if (element->IsBoundaryElement()) elementStart = "systemeElementStart";
+    if (element->IsMsElement()) elementStart = "systemeElementStart";
```

Comment:
(MINOR) Typo "systemElementStart"

Reply:
Another thing: When removing ids during the page based export, it should make sense to still keep the ids of milestone elements. Otherwise the reference `startid=#...` in `msEnd` does not really make sense. So `mdivb, secb` must be collected in the `FindAllReferencedObjects` functor.